src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -383,7 +383,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     locals = localBuilder.ToImmutableAndFree()
                 End If
 
-                statements.Add(New BoundReturnStatement(methodBlock.EndBlockStatement, New BoundLocal(methodBlock.EndBlockStatement, localForFunctionValue, isLValue:=False, type:=localForFunctionValue.Type), Nothing, Nothing))
+                statements.Add(New BoundReturnStatement(methodBlock.EndBlockStatement,
```

Comment:
Should we mark the return statement with end block syntax as implicit, so that empty sub methods have no explicit operations?

Reply:
> Should we mark the return statement with end block syntax as implicit, so that empty sub methods have no explicit operations?

This flag also affects how we emit PDB sequence points. Marking nodes here would affect debugging experience.